app/models/manageiq/providers/ibm_power_hmc/inventory/parser/infra_manager.rb

```diff
@@ -207,6 +207,37 @@ def parse_vm_advanced_settings(vm, lpar)
       :value        => lpar.ref_code,
       :read_only    => true
     )
+    unless lpar.proc_units.to_s.strip.empty?
+      persister.vms_and_templates_advanced_settings.build(
+        :resource     => vm,
+        :name         => 'entitled_processors',
+        :display_name => _('Entitled Processors'),
+        :description  => _('The number of entitled processors assigned to the VM'),
+        :value        => lpar.proc_units,
+        :read_only    => true
+      )
+    end
+
+    $ibm_power_hmc_log.info("#{self.class}##{__method__} : lpar.dedicated = #{lpar.dedicated}")
+    if lpar.dedicated.eql? "true"
```

Comment:
You could do this without duplicating all of the common values:

```suggestion
    proc_type = lpar.dedicated ==  "true" ? "dedicated" : lpar.sharing_mode
    persister.vms_and_templates_advanced_settings.build(
      :resource     =>vm,
      :name         => 'processor_type',
      :display_name => _('Processor type'),
      :description  => _('dedicated: Dedicated, shared: Uncapped shared, capped: Capped shared'),
      :value        => proc_type,
      :read_only    => true
    )
```